charts/govuk-rails-app/templates/deployment.yaml

```diff
@@ -31,8 +31,10 @@ spec:
             - name: metrics
               containerPort: {{ .Values.metricsPort }}
           envFrom:
+          {{- if .Values.govukAppsEnv.enabled }}
```

Comment:
It's been a while since I did this, but I think it was complaining about something not set which was why I have it as being optional. Could revisit it to see if there's a way to leave it in without causing the app to break

Reply:
Oh sorry, I see it - it's because we're installing this app in a different namespace, where that configmap won't exist. I don't mind which way we solve it for now - we can always iterate :)